extensions/smallrye-fault-tolerance/deployment/src/main/java/io/quarkus/smallrye/faulttolerance/deployment/SmallRyeFaultToleranceProcessor.java

```diff
@@ -13,18 +16,14 @@
 import org.eclipse.microprofile.faulttolerance.FallbackHandler;
 import org.eclipse.microprofile.faulttolerance.Retry;
 import org.eclipse.microprofile.faulttolerance.Timeout;
+import org.jboss.jandex.*;
 import org.jboss.jandex.AnnotationTarget.Kind;
-import org.jboss.jandex.ClassInfo;
-import org.jboss.jandex.DotName;
-import org.jboss.jandex.IndexView;
 
 import com.netflix.hystrix.HystrixCircuitBreaker;
 
-import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
-import io.quarkus.arc.deployment.AnnotationsTransformerBuildItem;
-import io.quarkus.arc.deployment.BeanContainerBuildItem;
-import io.quarkus.arc.processor.AnnotationsTransformer;
-import io.quarkus.arc.processor.BuiltinScope;
+import io.quarkus.arc.deployment.*;
```

Comment:
We should not use wildcard in imports...

Reply:
Even with the style settings the IDE keeps switching it to wildcard because there are so many.

Any suggestions?

Reply:
Well, eclipse has an option "number of imports needed for .*". I assume that other IDEs have similar settings?

Reply:
Ok, figured it out. Update pushed